Portfolio/index.html

```diff
@@ -33,12 +39,48 @@ <h2>Skills</h2>
         <li>React</li>
         <li>Node js</li>

```

Comment:
Consider adding proper class names for the social media icons for better styling and alignment.